rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -625,7 +631,9 @@ void game_list_frame::ShowContextMenu(const QPoint &pos)
 	QFont f = boot->font();
 	f.setBold(true);
 	boot->setFont(f);
-	QAction* configure = myMenu.addAction(tr("&Configure"));
+	QMenu* configurations_menu = myMenu.addMenu(tr("&Configurations"));
+	QAction* configure = configurations_menu->addAction(tr("&Configure"));
```

Comment:
Having "Configure" and "Configure pads" as the two options seems confusing. You should rename the other one to tell what it actually configures, for example:
```suggestion
	QAction* configure = configurations_menu->addAction(tr("&Configure settings"));
```